packages/cosmiconfig/test/getPropertyByPath.test.ts

```diff
@@ -0,0 +1,92 @@
+import { describe, test, expect } from 'vitest';
+import { getPropertyByPath } from '../src/util';
```

Comment:
I don't think this will work - the file does not seem to exist. We should just be able to delete this test file- it already exists in the core package.